stripe/src/main/java/com/stripe/android/paymentsheet/CurrencyFormatter.kt

```diff
@@ -9,19 +9,25 @@ import kotlin.math.pow
 internal class CurrencyFormatter {
     fun format(
         amount: Long,
-        currency: Currency
+        currencyAmount: Currency,
+        targetLocale: Locale = Locale.getDefault()
     ): String {
-        val currencySymbol = currency.getSymbol(Locale.getDefault())
+        val currencySymbol = currencyAmount.getSymbol(targetLocale)
         val majorUnitAmount =
-            amount / MAJOR_UNIT_BASE.pow(getDefaultFractionDigits(currency).toDouble())
-        val currencyFormat = NumberFormat.getCurrencyInstance()
+            amount / MAJOR_UNIT_BASE.pow(getDefaultFractionDigits(currencyAmount).toDouble())
+        val currencyFormat = NumberFormat.getCurrencyInstance(
+            Locale(
+                "",
```

Comment:
Can you add a comment as to why we're using an empty string for language?

Reply:
I don't have a good reason, except that appears how it is done.  See: https://github.com/stripe/stripe-android/blob/master/stripe/src/main/java/com/stripe/android/view/CountryUtils.kt